### PR TITLE
remove some #if's around symbol_print()

### PR DESCRIPTION
**Pull request by WalterBright**
don't actually need them

### CODE REVIEW

File: src/backend/elfobj.c [hunk 1]
```diff
@@ -3495,6 +3493,8 @@ void Obj::gotref(symbol *s)
     }
 }
 
+#endif
```
Comment (yebblies) [inline src/backend/elfobj.c]: Did you unintentionally revert https://github.com/WalterBright/dmd/commit/2c04a422e0ec2d5a291ce29fc19cff8706941c01 ?

Comment (WalterBright) [inline src/backend/elfobj.c]: no, that's right.

Comment (WalterBright) [inline src/backend/elfobj.c]: oh, crud!

**Comment by yebblies** [comment]
Auto-merge toggled on

**Pull request merged**